jumanji/environments/logic/minesweeper/utils.py

```diff
@@ -68,28 +73,28 @@ def get_mined_board(state: State) -> chex.Array:
 
 def explored_mine(state: State, action: chex.Array) -> chex.Array:
     """Check if an action is exploring a square containing a mine."""
-    height, width = action
-    index = width + height * state.board.shape[-1]
+    row, col = action
+    index = col + row * state.board.shape[-1]
     mined_board = get_mined_board(state=state)
     return mined_board[index] == IS_MINE
 
 
 def count_adjacent_mines(state: State, action: chex.Array) -> chex.Array:
     """Count the number of mines in a 3x3 patch surrounding the selected action."""
-    action_height, action_width = action
+    action_row, action_col = action
     mined_board = get_mined_board(state=state).reshape(
         state.board.shape[-2], state.board.shape[-1]
     )
```

Comment:
Would this be neater?
```suggestion
    mined_board = get_mined_board(state=state).reshape(*state.board.shape)
```